applications/serial_lte_modem/src/main.c

```diff
@@ -324,6 +330,22 @@ void handle_mcuboot_swap_ret(void)
 	switch (type) {
 	/** Attempt to boot the contents of slot 0. */
 	case BOOT_SWAP_TYPE_NONE:
+	/** MCUBOOT set BOOT_SWAP_TYPE_NONE after swapping B1 image, even TEST flag is set
+	 * by dfu_target_mcuboot library. There is no need to confirm image for B1 update.
+	 * But active slot is needed to be checked.
+	 */
```

Comment:
I understand this point as S0/S1 active status must be swapped after bootloader FOTA.
Applications like asset_tracker_v2 does not do the same.
Recommend to leave this as an inforamtion in the log, but not necessarily used to judge FOTA success.

Reply:
Are you saying that we don't check whether S0/S1 of B1 is swapped. Just judge it as success?
asset_tracker_v2 doesn't check app and bl result, they just confirm image after FOTA.

Reply:
Swapping S0/S1 active status (instead of image) is a good indication of a successful FOTA of MCUBoot image. By reaching this point, either S0 or S1 is working. Unlike MCUBoot APP FOTA, there is no possibility to rollback to previous B1 (if I'm correct) in the case SLM consider FOTA a failure. Thus I think a log of warning is enough.

Swap type check was added to SLM to confirm TEST OK, and was added with MCUBoot APP FOTA in mind only.